ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/filtered_tenants.py

```diff
@@ -57,6 +57,7 @@ def filtered_tenants(self: SharedUtils) -> list[dict]:
                     "vrfs": [
                         {
                             "name": "default",
+                            "wan_vni": 1,
```

Comment:
need to add `wan_vni: 1` in the else statement to ensure VRF default is always put on the WAN

Reply:
tested and works